pmemkv/pom.xml

```diff
@@ -30,6 +30,12 @@ LICENSE file.
   <name>PmemKV Binding</name>
   <packaging>jar</packaging>
 
+  <properties>
+    <!-- extra parameters to allow easier building with pmemkv from sources -->
+    <pmemkv.packageName>pmemkv-root</pmemkv.packageName>
+    <pmemkv.packageVersion>[1.0.1,)</pmemkv.packageVersion>
```

Comment:
I think we can require at least 1.1.0 (before that, the performance is tragic)

Reply:
+1

Reply:
true that, mea culpa